packages/plugin-react/src/index.ts

```diff
@@ -47,6 +47,14 @@ export interface Options {
   babel?:
     | BabelOptions
     | ((id: string, options: { ssr?: boolean }) => BabelOptions)
+  /**
+   * React refresh runtime url prefix.
+   * Useful in module federation context to enable HMR by
+   * setting the host url on a vite config which is serving a remote app.
+   * @example
+   * reactRefreshPrefix: 'http://localhost:3000'
+   */
+  reactRefreshPrefix?: string
```

Comment:
While technically you could use it to have only a path prefix, `host` makes moire sense but the intended usage 
```suggestion
  reactRefreshHost?: string
```